relational_operators/WorkOrder.proto

```diff
@@ -83,7 +84,8 @@ message DeleteWorkOrder {
 message DestroyHashWorkOrder {
   extend WorkOrder {
     // All required.
-    optional uint32 join_hash_table_index = 112;
+    optional int32 relation_id = 112;
```

Comment:
We should serialize `PartitionSchemeHeader` instead of `relation_id`. Also, this entry is not required, so move it after `join_hash_table_group_index`.